lib/l10n/app_it.arb

```diff
@@ -0,0 +1,260 @@
+  "load_more": "Carica altro",
+  "playlists": "Playlist",
+  "artists": "Artisti",
+  "albums": "Album",
+  "tracks": "Brani",
+  "downloads": "Download",
+  "filter_playlists": "Filtra le tue playlist...",
+  "liked_tracks": "Brani Preferiti",
+  "liked_tracks_description": "Tutti i tuoi brani preferiti",
+  "create_playlist": "Crea Playlist",
+  "create_a_playlist": "Crea una playlist",
+  "create": "Crea",
+  "cancel": "Annulla",
+  "playlist_name": "Nome Playlist",
+  "name_of_playlist": "Nome della playlist",
+  "description": "Descrizione",
+  "public": "Pubblico",
+  "collaborative": "Collaborative",
+  "search_local_tracks": "Cerca tra i brani locali...",
+  "play": "Riproduci",
+  "delete": "Elimina",
+  "none": "Nessuno",
+  "sort_a_z": "Ordina da A-Z",
+  "sort_z_a": "Ordina da Z-A",
+  "sort_artist": "Ordina per Artista",
+  "sort_album": "Ordina per Album",
+  "sort_tracks": "Ordina Brani",
+  "currently_downloading": "Attualmente in Downloading ({tracks_length})",
+  "cancel_all": "Annulla Tutto",
+  "filter_artist": "Filtra artisti...",
+  "followers": "{followers} Follower",
+  "add_artist_to_blacklist": "Aggiungi artista a blacklist",
+  "top_tracks": "Migliori Brani",
+  "fans_also_like": "Ai Fans piace anche",
+  "loading": "Caricamento...",
+  "artist": "Artista",
+  "blacklisted": "Blacklisted",
+  "following": "Following",
+  "follow": "Follow",
+  "artist_url_copied": "URL dell'Artista copiato nella clipboard",
+  "added_to_queue": "Aggiunto {tracks} alla coda",
+  "filter_albums": "Filtra album...",
+  "synced": "Sincronizzato",
+  "plain": "Plain",
+  "shuffle": "Shuffle",
+  "search_tracks": "Cerca brani...",
+  "released": "Rilasciato",
+  "error": "Errore {error}",
+  "title": "Titolo",
+  "time": "Time",
+  "more_actions": "Altre azioni",
+  "download_count": "Download ({count})",
+  "add_count_to_playlist": "Aggiungi ({count}) alla Playlist",
+  "add_count_to_queue": "Aggiungi ({count}) alla Coda",
+  "play_count_next": "Riproduci ({count}) seguenti",
+  "album": "Album",
+  "copied_to_clipboard": "Copiato {data} nella clipboard",
+  "add_to_following_playlists": "Aggiungi {track} alle Playlist successive",
+  "add": "Aggiungi",
+  "added_track_to_queue": "Aggiunta {track} alla coda",
+  "add_to_queue": "Aggiungi alla coda",
+  "track_will_play_next": "{track} verrà riprodotta dopo",
+  "play_next": "Riproduci seccessivo",
+  "removed_track_from_queue": "Rimossa {track} dalla coda",
+  "remove_from_queue": "Rimuovi dalla coda",
+  "remove_from_favorites": "Rimuovi dai preferiti",
+  "save_as_favorite": "Salva come preferito",
+  "add_to_playlist": "Aggiungi alla playlist",
+  "remove_from_playlist": "Rimuovi dalla playlist",
+  "add_to_blacklist": "Aggiungi alla blacklist",
+  "remove_from_blacklist": "Rimuovi dalla blacklist",
+  "share": "Condividi",
+  "mini_player": "Mini Player",
+  "slide_to_seek": "Slide to seek forward or backward",
```

Comment:
@OpenCode what about it ?

Reply:
The correct translation should be "Scorrere per cercare in avanti o indietro". But I don't know the context.